ovos_backend_client/version.py

```diff
@@ -2,6 +2,6 @@
 # START_VERSION_BLOCK
 VERSION_MAJOR = 0
 VERSION_MINOR = 1
-VERSION_BUILD = 0
-VERSION_ALPHA = 0
+VERSION_BUILD = 2
+VERSION_ALPHA = 1
```

Comment:
Looks like a PR was merged to `dev` after this PR was opened. Since that's already pushed `0.1.2a1`, I think the best option is to propose an `0.1.2` stable release so the stable release is newer than that last alpha